simpleSequential/andFunction.py

```diff
@@ -3,3 +3,5 @@
 # Ex: 
 #   and(True, False) return False
 #   and(True, True)  return True
+def andFunction(boo1: bool, boo2: bool) -> bool:
```

Comment:
Can you use `bool1` and `bool2` instead? I never see anyone in the industry use `boo`. In fact, people don't use `bool` either. Let use `var1` and `var2`,

Reply:
acknowledge.